slog/sentryslog.go

```diff
@@ -42,9 +44,17 @@ var (
 )
 
 type Option struct {
-	// Level sets the minimum log level to capture and send to Sentry.
-	// Logs at this level and above will be processed. The default level is debug.
+	// Deprecated: Use EventLevel instead. Level is kept for backwards compatibility and defaults to EventLevel.
 	Level slog.Leveler
+	// EventLevel sets the minimum log level to capture and send to Sentry as an Event.
+	// Logs at this level and above will be processed as events.
+	// Defaults to slog.LevelError.
+	EventLevel slog.Leveler
+
+	// LogLevel sets the minimum log level to capture and send to Sentry as a Log entry.
+	// Logs at this level and above will be processed as log entries.
+	// Defaults to slog.LevelDebug.
+	LogLevel slog.Leveler
```

Comment:
I know I said previously that `EnableLogs` gating this is fine, but I tried running this on a local go app and now I'm now debating if there should be a way to turn this off.

I can see users wanting to disable `EventLevel` completely and only rely on `LogLevel` for example.

Maybe we expose separate boolean options? What do you think?